addons/account/static/src/scss/account_searchpanel.scss

```
@@ -13,6 +13,10 @@
         header {
             margin-left: 0;
         }
+        &:first-child .o_toggle_fold
+        {
+            width: 0;
+        }
```

Wouldn't it be better to check for the emptiness of the node instead?
> If there is a caret in there, take some space; if not, don't.
```suggestion
        .o_toggle_fold:empty {
            width: 0;
        }
```

---

this works, it seems to me a better solution. I think I need a sass course :worried: .